components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -111,12 +119,6 @@ function TeamUsage() {
         return inMinutes + " min";
     };
 
-    const calculateTotalUsage = () => {
-        let totalCredits = 0;
-        billedUsage.forEach((session) => (totalCredits += session.credits));
-        return totalCredits.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 });
-    };
-
     const handleMonthClick = (start: any, end: any) => {
         setStartDateOfBillMonth(start);
         setEndDateOfBillMonth(end);
```

Comment:
This doesn't yet trigger a reload and rerender of usage data

Reply:
indeed, thanks!

Reply:
Actually it does the request which fails with:
> Request listBilledUsage failed with message: 13 INTERNAL: unable to retrieve billed usage

Reply:
> 	usageRecords, total, totalCreditsUsed, err := db.ListUsage(ctx, s.conn, db.AttributionID(in.GetAttributionId()), from, to, order, offset, limit)

`sql.NullFloat64` it should be